.eslintrc

```diff
@@ -10,7 +10,12 @@
     }
   },
   "rules": {
-    "cypress/no-unnecessary-waiting": "off"
+    "cypress/no-unnecessary-waiting": "off",
+    "eqeqeq": "error",
+    "array-callback-return": "error",
+    "react-hooks/exhaustive-deps": 0,
```

Comment:
We have like 30 warnings for this. I agree that some will need fixing but sometimes this check is very strict and the code does not need to include all deps. The intention of this pr was to have a clean build without warns/errors to avoid introducing new ones. What I can do is left this set to `warn` and disable each current warning individually and add a todo on each. Or just left the warning. What do you think?

Reply:
After discussing this we are adding individual eslint-disable on each warning